src/System.Net.Security/src/System/Net/SecureProtocols/_FixedSizeReader.cs

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+namespace System.Net
+{
+    //
+    // Reads a fixed size packet from a stream, can disover EOF as 0 bytes read from stream.
+    //
+    internal class FixedSizeReader
+    {
+        private static readonly AsyncCallback s_readCallback = new AsyncCallback(ReadCallback);
+
+        // TODO (Issue #3114): Implement this using TPL instead of APM.
+        private readonly StreamAsyncHelper _TransportAPM;
+        private readonly Stream _Transport;
+        private AsyncProtocolRequest _Request;
+        private int _TotalRead;
+
```

Comment:
All of these fields are _PascalCased instead of _camelCased.  Is that something that CodeFormatter missed, or was it intentional?